src/main/kotlin/ar/edu/unahur/obj2/servidorWeb/Analizador.kt

```diff
@@ -0,0 +1,12 @@
+package ar.edu.unahur.obj2.servidorWeb
+
+abstract class Analizador() {
+    val respuestasRecividas = mutableListOf<Respuesta>()
```

Comment:
Ups, no puedo con mi genio:
```suggestion
    val respuestasRecibidas = mutableListOf<Respuesta>()
```